.github/workflows/documentation.yml

```diff
@@ -0,0 +1,33 @@
+name: Build documentation
+on:
+  workflow_dispatch:
```

Comment:
I haven't looked at this one, does it cover the specific workflow file in some way? My workflow just added the workflow file as another path condition, though I'm not sure how that affects `pull_request`, I think the docs cover that concern that changes to the workflow from a PR won't be applied and need merging first to trigger.

Reply:
You mean workflow_dispatch? That enables you to manually trigger the workflow without pushing anything.